theano/tensor/signal/pool.py

```diff
@@ -403,7 +410,7 @@ def c_code(self, node, name, inp, out, sub):
             }
             else
             {
-                z_r = std::max(0, (r - 1 - %(ds0)s) / %(st0)s + 1) + 1;
+                z_r = std::max(0, (r - 1 - %(ds0)s + %(st0)s) / %(st0)s) + 1;
```

Comment:
Looks good, that's what we have in Lasagne as well. But wouldn't this one need to be changed as well? https://github.com/nouiz/Theano/blob/ecc1b37773c284c2dc9e2b49aecc29ecb3f3ac0e/theano/tensor/signal/pool.py#L223 (Shape inference seemed to be correct, so I guess not, but why?)